src/pages/UpdateServers.js

```diff
@@ -254,6 +254,10 @@ class UpdateServers extends BaseUpdateWizardPage {
   }
 
   async getServerStatuses() {
+    //we need the model from the props here, we will try to wait for them to show up
+    while(!this.props.model || !this.state.internalModel) {
```

Comment:
We should not be checking for `this.props.model` here.  The `setState` call that we are waiting for does not populate this.props -- react populates this before `componentDidMount` is called.

Reply:
This issue is happening because the code attempts to perform some computation on `this.props.model` which is not populated in this component. I think what we need to do is pass in a promise for when the model will be loaded so that I can await for it.